tests/opentitan/opentitan-210214/module_tests/otp_ctrl_part_buf/Makefile.in

```diff
@@ -0,0 +1,24 @@
+include $(TEST_DIR)/../Makefile.in
+TOP_FILE := \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_abstract_prim_pkg_0.1/prim_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_util_0.1/rtl/prim_util_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_lc_ctrl_state_pkg_0.1/rtl/lc_ctrl_state_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_lc_ctrl_pkg_0.1/rtl/lc_ctrl_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_otp_ctrl_pkg_0.1/rtl/otp_ctrl_reg_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_otp_ctrl_pkg_0.1/rtl/otp_ctrl_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_otp_ctrl_pkg_0.1/rtl/otp_ctrl_part_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_otp_pkg_0.1/rtl/prim_otp_pkg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_secded_0.1/rtl/prim_secded_72_64_enc.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_secded_0.1/rtl/prim_secded_72_64_dec.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_otp_ctrl_0.1/rtl/otp_ctrl_ecc_reg.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_generic_buf_0/rtl/prim_generic_buf.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_abstract_buf_0/prim_buf.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_generic_flop_0/rtl/prim_generic_flop.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_xilinx_flop_0/rtl/prim_xilinx_flop.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_prim_abstract_flop_0/prim_flop.sv \
+    $(OPENTITAN_BUILD)/src/lowrisc_ip_otp_ctrl_0.1/rtl/otp_ctrl_part_buf.sv
+
+INCLUDE := -I$(OPENTITAN_BUILD)/src/lowrisc_prim_assert_0.1/rtl/
+
+TOP_MODULE := otp_ctrl_part_buf
+VERILATOR_FLAGS := -Wno-UNOPTFLAT -Wno-LITENDIAN $(TEST_DIR)/otp_ctrl_part_buf.vlt
```

Comment:
Is the `LITENDIAN` warning present in original Verilator or does it come from UHDM? In any case, it would be good to note what's causing it like in the .vlt file.

Reply:
It is in original verilator. I don't know if it is in UHDM (it rather should be), because the other error comes first.
I tried to find the values that are passed to the parameters of the instance, but it is rather hard.

I would leave this test with the default parameter values. If the errors related to values happen, I will prepare test of the upper module (otp_ctrl).